login.php

```diff
@@ -62,9 +66,14 @@
         $name = $_POST['name'];
         $password = $_POST['password'];
 
+		$hashedpw= $db->getRow("SELECT `password` FROM `".MLS_PREFIX."users` WHERE `username` = ?s", $_POST['name']);
 
-        if(!($usr = $db->getRow("SELECT `userid` FROM `".MLS_PREFIX."users` WHERE `username` = ?s AND `password` = ?s", $name, sha1($password))))
-            $page->error = "Username or password are wrong !";
+        if(!($usr = $db->getRow("SELECT `userid` FROM `".MLS_PREFIX."users` WHERE `username` = ?s", $_POST['name'])) || !(password_verify($password, ($hashedpw->password)))){            
+	     $page->error = "Username or password are wrong !";
+		}
+		else if($_POST)
+			if(!$captcha)
+				$page->error = "Invalid captcha code !";
         else {
             if($_POST['r'] == 1){
                 $path_info = parse_url($set->url);
```

Comment:
I was referring to the `remember` me functionality. it saves the hashed password in a cookie. look at the lines below this.